subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -29,14 +29,19 @@
 #include "lll.h"
 #include "lll_clock.h"
 #include "lll_conn.h"
+#include "lll_conn_iso.h"
+#include "lll_tim_internal.h"
```

Comment:
This include seems to be obsolete since f9511079b2737485494534c7efce9b9fdb12e9ca

Reply:
Oh, bad rebase merge. Thanks